src/emails/templates/monthlyActivity/MonthlyActivityEmail.stories.tsx

```diff
@@ -10,20 +10,31 @@ import { SanitizedSubscriberRow } from "../../../app/functions/server/sanitize";
 import { getL10n } from "../../../app/functions/l10n/storybookAndJest";
 import { DashboardSummary } from "../../../app/functions/server/dashboard";
 
+type StoryProps = Props & { emulateDarkMode?: boolean };
 const meta: Meta<FC<Props>> = {
   title: "Emails/Monthly activity",
-  component: (props: Props) => (
-    <StorybookEmailRenderer>
-      <MonthlyActivityEmail {...props} />
-    </StorybookEmailRenderer>
+  component: (props: StoryProps) => (
+    <>
+      {props.emulateDarkMode && (
+        <style>{`
+        * {
+          background-color: #1e293b !important;
+          color: white !important;
+        }
+      `}</style>
+      )}
```

Comment:
Question (non-blocking): Would it make sense to move this into `StorybookEmailRenderer` so the prop `emulateDarkMode` can be used in all stories without duplicating the styles?

Reply:
That's an excellent idea. Stories will still have to pass on the prop, _but_ I also added support for the `prefers-color-scheme` media query, so you can now set your browser to dark mode for every email story. Added in 24fe12f49.

Reply:
Awesome, thanks!

Reply:
Haha, and of course now it's complaining that this isn't covered by tests :sweat_smile: